code/game/objects/random/random.dm

```diff
@@ -11,11 +11,9 @@
 	..()
 	if (!prob(spawn_nothing_percentage))
 		spawn_item()
-
-/obj/random/initialize()
```

Comment:
I'm told that apparently initialize() is supposed to be called by the base New() proc, but for some reason that isn't happening with these items. My way works, the old way doesn't work, and I don't know why.